frontend/public/metalkube/components/host/host-detail.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Row, Col } from 'patternfly-react';
+import { getResource } from 'kubevirt-web-ui-components';
+
+import { BaremetalHostModel } from '../../models/host';
+import { navFactory } from '../utils/okdutils';
+import { WithResources } from '../../../kubevirt/components/utils/withResources';
+import { DetailsPage, List, ListHeader, ColHead, ResourceRow } from '../factory/okdfactory';
+
+
+// TODO(honza): the BaremetalHostModel component should go into web-components once it's stable
```

Comment:
please remove

Right, we plan to share the model definitions. Anyway, they will still be imported from a local .js file, i.e. via reexporting.

Reply:
Sorry, "model" was the wrong word. I meant the detail component just below. Comment removed none the less.